config/environments/production.rb

```diff
@@ -96,16 +96,6 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  # Datadog API client used to send exception data to Datadog.
-  datadog_api_config = Rails.application.secrets.datadog
-  if datadog_api_config[:api_enabled]
-    datadog_api_client = Dogapi::Client.new(datadog_api_config[:api_key], datadog_api_config[:application_key])
-    config.middleware.use ExceptionNotification::Rack,
-    datadog: {
-      client: datadog_api_client
-    }
-  end
```

Comment:
I've moved this to its own initializer and enabled it in dev & test environments. This ensures that if there are any issues with the configuration, we'll find out about it during CI runs. (No data will be sent to Datadog unless the developer adds legitimate keys to `config/secrets.yml`.)